src/proto/message_translation/tbots_geometry_test.cpp

```diff
@@ -52,3 +55,14 @@ TEST(TbotsProtobufTest, polygon_msg_test)
     EXPECT_EQ(polygon.getPoints()[2].x(), polygon_msg->points(2).x_meters());
     EXPECT_EQ(polygon.getPoints()[2].y(), polygon_msg->points(2).y_meters());
 }
+
+TEST(TbotsProtobufTest, circle_msg_test)
+{
+    auto circle_1     = Circle(Point(-1, 1), 4);
+    auto circle_msg_1 = createCircleProto(circle_1);
+    auto circle_2     = createCircle(*circle_msg_1);
+    auto circle_msg_2 = createCircleProto(circle_2);
+
+    EXPECT_TRUE(
+        google::protobuf::util::MessageDifferencer::Equals(*circle_msg_1, *circle_msg_2));
```

Comment:
this isn't a perfect test because you could have:
~~~
circle_1 = Circle(1,1, radius=3)
circle_1_msg = createCircleProto(circle_1) // CircleMsg(1,1, radius=0) drops the radius value
circle_2 = createCircle(circle_1_msg) // Circle(1,1, radius=0) radius value is still 0
circle_2_msg = createCircleProto(circle_2) // CircleMsg(1,1, radius=0) drops the radius value

EXPECT_TRUE(circle_1_msg, circle_2_msg) is true but we've dropped the radius
~~~
probably worth checking if circle_1 == circle_2 ill add that too

Reply:
I've made a ticket here to track this work for a later day:
https://github.com/UBC-Thunderbots/Software/issues/2403

Reply:
Aren't both radius and origin translated in both translation functions? Why would radius be dropped?